deps.qt/checksums/qtwinextras-everywhere-src-5.15.2.tar.xz.sha256

```diff
@@ -0,0 +1 @@
+65b8272005dec00791ab7d81ab266d1e3313a3bbd8e54e546d984cf4c4ab550e  qtwinextras-everywhere-src-5.15.2.tar.xz
```

Comment:
This would be needed for the Windows build if we end up downloading the individual component archives. Currently, I'm still working against the git super repo, so you can omit this from this PR.